build/kubectl-testkube/Dockerfile

```diff
@@ -2,6 +2,7 @@
 ARG ALPINE_IMAGE
 FROM ${ALPINE_IMAGE}
 COPY kubectl-testkube /bin/kubectl-testkube
-RUN mkdir /.testkube && echo "{}" > /.testkube/config.json && chmod -R 755 /.testkube && chown -R 1001:1001 /.testkube && chmod 660 /.testkube/config.json
+COPY configure.sh /configure.sh
+RUN ln -s /bin/kubectl-testkube /bin/testkube && mkdir /.testkube && echo "{}" > /.testkube/config.json && chmod -R 755 /.testkube && chown -R 1001:1001 /.testkube && chmod 660 /.testkube/config.json && chmod +x /configure.sh
 USER 1001
-ENTRYPOINT ["/bin/kubectl-testkube"]
+ENTRYPOINT ["/configure.sh"]
```

Comment:
With that change we should update the docs for `testkube-cli` in `testkube-cli-docker.md`